library/src/main/java/com/datatorrent/lib/io/jms/JMSBase.java

```diff
@@ -112,14 +156,14 @@ public Destination getDestination()
     return destination;
   }
 
-  public String getConnectionFactoryClass()
```

Comment:
Leave the get/set connection factory class helper methods and check for DefaultConnectionFactoryBuilder like below for the properties.